data/sql/updates/pending_db_world/rev_1601868704763174600.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601868704763174600');
+
+DELETE FROM `gameobject` WHERE (`id` = 182947) AND (`guid` IN (23860));
```

Comment:
this query may fail if there is already a row having `guid` = `23860` and `id` different than `182947`.

If you want to make sure the query doesn't fail, then:

```sql
DELETE FROM `gameobject` WHERE `guid` = 23860;
```

Also, if (and ONLY if) you want to make sure your new `INSERT` will be the ONLY spawn of the gameobject `182947`, you can add this too:

```sql
DELETE FROM `gameobject` WHERE `id` = 182947;
```

resulting in:

```sql
DELETE FROM `gameobject` WHERE `guid` = 23860;
DELETE FROM `gameobject` WHERE `id` = 182947; -- only if you want to delete all current spawns of 182947
```

Reply:
Would it be better to have something like this?
```sql
 UPDATE `gameobject` SET `spawnMask` = 3 WHERE (`guid` = 23860 and `map` = 555);
```
As we only need the spawnMask of 23860 to be updated in the map 555 (Auchindoun: Shadow Labyrinth).

FYI: The generated query i added in the `.sql` was generated by Keira3

Reply:
ah yes, definitely. If you are updating an existing row, always prefer `UPDATE`

Reply:
Perfect :D